config/config.go

```diff
@@ -664,6 +671,33 @@ func (c *KubernetesSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) er
 	return checkOverflow(c.XXX, "kubernetes_sd_config")
 }
 
+// EC2SDConfig is the configuration for EC2 based service discovery.
+type EC2SDConfig struct {
+	Region          string             `yaml:"region"`
+	AccessKey       string             `yaml:"access_key,omitempty"`
+	SecretKey       string             `yaml:"secret_key,omitempty"`
```

Comment:
He was referring to this regex:

https://github.com/prometheus/prometheus/blob/ec04ba38a2672786f69455b735fd43e6d06afa90/config/config.go#L36

...which is used by the status page config printer to "blur" out credentials.